src/ray/raylet/worker_pool.cc

```diff
@@ -77,6 +77,8 @@ WorkerPool::WorkerPool(instrumented_io_context &io_service, const NodeID node_id
       periodical_runner_(io_service),
       get_time_(get_time) {
   RAY_CHECK(maximum_startup_concurrency > 0);
+  // A no-op so the metric is 0 but exists.
+  stats::NumWorkersStarted.Record(0);
```

Comment:
Can you comment why we need it ?

Reply:
Wait there is a comment. What type of description are you looking for?

Reply:
I don't understand why metric with 0 value should exist?

Reply:
i think the unit tests only launch tasks on the head node, so the metric otherwise wouldn't exist on the second node.

Reply:
Wait I don't see you updated the comment... Let's not have mysterious code with vague comment like this.

Reply:
Oh sorry. I saw you updated the comment. My bad lol